packages/custom-functions-metadata/test/src/testCases.ts

```diff
@@ -48,7 +48,7 @@ describe("test cases", function() {
                         const expectedErrorsFile = path.join(testCaseDirPath, `expected.${scriptType}.errors.txt`);
                         const actualMetadataFile = path.join(testCaseDirPath, `actual.${scriptType}.json`);
                         const expectedMetadataFile = path.join(testCaseDirPath, "expected.json");
-                        const expectedMetadata: string | undefined = readFileIfExists(expectedMetadataFile) || "";
+                        const expectedMetadata = readFileIfExists(expectedMetadataFile) || "";
```

Comment:
Why are we removing this explicit variable assign?

Reply:
I looked over the diff and noticed that the `| undefined` should be removed. But rather than leave `: string`, I decided to just remove it.

I would say that in general, it is better to put the types on variables. 

In this case, because the other variables don't have it, I decided to deviate from this.

Reply:
Makes sense